Detectors/EMCAL/simulation/include/EMCALSimulation/SimParam.h

```diff
@@ -91,6 +100,9 @@ class SimParam
   Double_t mTimeResolutionPar0{ 0.26666 }; // Time resolution of FEE electronics
   Double_t mTimeResolutionPar1{ 1.4586 };  // Time resolution of FEE electronics
   Int_t mNADCEC{ 0x10000 };                // number of channels in EC section ADC
+  Float_t mTimeResponseTau{ 2.35 };        // Raw time response function tau parameter
+  Float_t mTimeResponsePower{ 2 };         // Raw time response function power parameter
+  Float_t mTimeResponseThreshold{ 0.001 }; // Raw time response function energy threshold
```

Comment:
There seems to be a mix of float and double throughout the digitization. Would it be possible to consistently use on or the other (maybe just float)?